js/application/init.js

```diff
@@ -194,7 +194,12 @@ define([
           document.title = this.config.title;
 
           // Launch the app specifics
-          this.main.start(this.config, dom.byId("viewDiv"), view, webmap, null, null);
+          this.main.start({
+            config: this.config,
+            viewDiv: dom.byId("viewDiv"),
+            view: view,
+            mapOrScene: webmap
```

Comment:
can we separate these out into `webmap` and `webscene` properties? Then users don't have to try to figure out which it is. and using it is easy as `object.webmap || object.webscene`